### DISCUSSION TITLE
Business ideas: New product, heat pump controller

### DISCUSSION THREAD

**Discussion by tux-** [UtilitechAS/amsreader-firmware#668] [Ideas]
I have been tinkering around with an ESP controller for my heat pump. Starting with a breadboard and a bit unstable connections, to finally having designed a pcb and soldered components on tight and neatly so the controller can fit inside the machine.

I was thinking maybe it would be a good idea to sell the hardware premade (code is open source on github already). I am not going into the business! But, if you want to expand, maybe it would be a possible expansion point :)

**Comment by ArnieO** [comment]
Hi Tux,

Thank you for a potentially interesting idea - but I have no idea what kind of inputs and outputs a typical heat pump system would need. 
Is your unit an air/air heatpump, or a more complex system?
And the different brands probably do not have the same type of inputs/outputs?

**Reply by tux-** [reply]
Here is a link to the software part (I can not find anyoue out there that offers hardware): https://github.com/gysmo38/mitsubishi2MQTT - So yes, it would not cover all possible heat pumps, but at least one type that is very common.

If you want pictures / the pcb files I created for the prints I finally got, etc, do not hessitate to contact me. Not sure if I can upload here or not.

**Reply by ArnieO** [reply]
I could as a start take a look at the PDF of your el. schematic for the board. 
Can you email it to me on post@amsleser.no?

**Reply by tux-** [reply]
Do not expect professionally made stuff here from me, but yeah, I'll be happy to send you what I have! Hopefully it can at least give a boost start of what would be required to make it work. But yeah, my tinkering is very, very far from anyting ready to be released. Would for sure need your expertise in electronics to be fine tuned and commercialized. Mail sent :)
